### PR TITLE
Les 454 created at

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

**Review by fooooxmr — APPROVED** [review]

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ def update
   private
 
   def category_params
-    params.permit(:name, :description, :status)
+    params.permit(:name, :description, :status, :created_at)
```
Comment (evrusha) [inline app/controllers/categories_controller.rb:44]: Why we need `:created_at` here? Active Record automatically timestamps create operation.

Comment (ritamma) [reply app/controllers/categories_controller.rb:44]: Ok,  I'll remove it

Comment (ritamma) [reply app/controllers/categories_controller.rb]: @evrusha Done

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -16,4 +16,8 @@ def show
   def user_find
     @user = User.find_by(id: params[:id])
   end
+
+  def user_params
+    params.permit(:name, :description, :avatar, :avatar_url, :gender, :birthday, :phone)
+  end
```
Comment (evrusha) [inline app/controllers/users_controller.rb:22]: Where is this method used?

**Review by evrusha — APPROVED** [review]

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (ritamma) [reply app/controllers/users_controller.rb]: This is already added in a branch with an avatar download for users, it's necessary to remove from this pull request. I've already done it.

**Review by hannrywislou — APPROVED** [review]

**Pull request merged**